backend/src/main.ts

```diff
@@ -11,11 +11,11 @@ import {
   CreateResponse,
   ChangesRequest,
   ChangesResponse
-} from "../../api/rbt/thirdparty/prosemirror//v1/authority_rbt.js";
+} from "../../api/reboot_prosemirror//v1/authority_rbt.js";
```

Comment:
Not your bug but can you remove the double `/`?
```suggestion
} from "../../api/reboot_prosemirror/v1/authority_rbt.js";
```